src/api/types/mobility.ts

```diff
@@ -1,4 +1,5 @@
 import {z} from 'zod';
+import {FormFactor} from '@atb/api/types/generated/mobility-types_v2';
```

Comment:
The FormFactor from Entur's qr-endpoint in Shared Mobility is the same as FormFactor from their Mobility API. However, we may still consider creating a separate type `ShmoFormFactor` just to separate them in case they diverge 🤔 
I'm thinking just keep it like this for now, but feel free to voice any opinion on this!